src/Crypto/core-crypto.asd

```diff
@@ -37,16 +37,16 @@ THE SOFTWARE.
                 (:file "hash")
                 (:file "ctr-hash-drbg")
                 (:file "primes")
-                #+:COM.RAL (:file "crypto-le")
-                (:file "kdf")
-                (:file "gf-571")
+                #+(AND :MACOSX :COM.RAL) (:file "crypto-le")
```

Comment:
It would be nice to remove or start deprecating the use of the `:com.ral` for `cl:*features*`.  

As it's use needs to be explained to every new developer, and isn't clear what it means.

Reply:
yes started thinking about that last night as I went weeding... Perhaps time to put a clear separation between my other uses of the crypto libs

Reply:
Again, not necessary for this pull request, but should be done sooner rather than later.